src/pages/Validator/AutocompleteList.js

```diff
@@ -2,36 +2,78 @@ import React from "react";
 
 export default function AutocompleteList({
   entities = {},
+  showAttributes,
   inputVal,
   onSelect,
   onCreate,
```

Comment:
Since we are also autocompleting attributes, it'd be nice to be specific with this one. Maybe rename it to something like `onCreateEntity`

Reply:
I agree, nice idea